pkg/utils/instance/instance_types.go

```diff
@@ -0,0 +1,8221 @@
+// / Generated by `ec2geninfo`
+
+package instance
+
+var InstanceTypesMap = map[string]InstanceInfo{}
+
+func init() {
+	for _, instance := range InstanceTypes {
+		InstanceTypesMap[instance.InstanceType] = instance
+	}
+}
+
+type InstanceInfo struct {
+	InstanceType             string
+	InstanceStorageSupported bool
+	EFASupported             bool
+	NvidiaGPUSupported       bool
+	NvidiaGPUType            string
+	NeuronSupported          bool
+	NeuronGPUType            string
```

Comment:
```suggestion
	NeuronDeviceType            string
```

Ref https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/arch/neuron-hardware/neuron-devices.html#neurondevices-arch

Reply:
Done